packages/patternfly-4/react-core/src/demos/PaginationTable.md

```diff
@@ -168,6 +168,9 @@ class ComplexPaginationTableDemo extends React.Component {
 }
 ```
 
+The below example illustrates the `defaultToFullPage` prop, which makes the following changes when the user sets the number of items to display per page to an amount that exceeds the remaining amount of data:
+- The component automatically changes the page back to the last full page of results, rather than defaulting to the final page of results.
```

Comment:
>defaulting to the final page of results

I wasn't quite sure what "final page" means or what its impact on the overall interaction would be. What's the default experience when the `defaultToFullPage` prop is omitted?

Reply:
@seanforyou23 in the default experience the last page will be an incomplete set of results, but I don't know that "incomplete" makes sense until you already understand how it will look.  I thought about typing out an example, but am not sure that putting more words towards it will reduce the confusion - perhaps adding steps for the user to follow in the demo and then pointing out the difference would be a clearer option.

Reply:
@seanforyou23 I've updated the demo docs based on my feedback above, please let me know if this makes it any better from your perspective.